modules/admin_manual/pages/configuration/server/occ_commands/core_commands/_2fa_core_commands.adoc

```diff
@@ -0,0 +1,49 @@
+= Two-Factor Authentication
+:page-noindex: yes
+
+The following commands only enable or disable the two-factor authentication for a particular user. See the xref:two-factor-totp[Two-Factor TOTP] section for managing the two-factor app provided by ownCloud.
+
+If a two-factor provider app is enabled, it is enabled for all users by default, though the provider can decide whether or not the user has to pass the challenge. In the case a user is losing access to the second factor like a lost phone with two-factor SMS verification, the admin can temporarily disable the two-factor check for that user via the occ command. After fixing the issue, the admin can reenable twofactor authentication for that user.
```

Comment:
```suggestion
If a two-factor provider app is enabled, it is enabled for all users by default, though the provider can decide whether or not the user has to pass the challenge. In case a user loses access to the second factor (e.g. because of a lost phone and two-factor SMS verification), the admin can temporarily disable the two-factor check for that user via the occ command. After fixing the issue, the admin can re-enable two-factor authentication for that user.
```